packages/web/src/components/table/Table.tsx

```diff
@@ -518,20 +528,14 @@ export const Table = ({
   // TODO: This is supposed to return a promise that resolves when the row data has been fetched.
   // It currently does not, but there are no issues with this currently so will fix if issues pop up
   const loadMoreRows = useCallback(
-    async ({
-      startIndex,
-      stopIndex
-    }: {
-      startIndex: number
-      stopIndex: number
-    }) => {
+    async ({ startIndex }: { startIndex: number }) => {
       if (!debouncedFetchMore) return null
       const offset = startIndex
-      const limit = stopIndex - startIndex + 1
+      const limit = fetchBatchSize
```

Comment:
the `stopIndex` passed from the `InfiniteLoader` to this fn is not correct and is not meant to be used here unless the `totalRowCount` is always known.  Using `fetchBatchSize` (the batch size specified by the parent) as the limit will always be correct.